<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>988</width>
    <height>849</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QLabel" name="first_view_description">
          <property name="minimumSize">
           <size>
            <width>400</width>
            <height>20</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>20</height>
           </size>
          </property>
          <property name="text">
           <string>Фронтальный срез</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="SliceLabel" name="first_view">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>400</width>
            <height>300</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="scaledContents">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QLabel" name="third_view_description">
          <property name="minimumSize">
           <size>
            <width>400</width>
            <height>20</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>20</height>
           </size>
          </property>
          <property name="text">
           <string>Корональный срез</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="SliceLabel" name="third_view">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>400</width>
            <height>300</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="scaledContents">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="QLabel" name="second_view_description">
          <property name="minimumSize">
           <size>
            <width>400</width>
            <height>20</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>20</height>
           </size>
          </property>
          <property name="text">
           <string>Сагиттальный срез</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="SliceLabel" name="second_view">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>400</width>
            <height>300</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="scaledContents">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>988</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="main_menu">
    <property name="title">
     <string>Меню</string>
    </property>
    <addaction name="action_open_dicom"/>
   </widget>
   <widget class="QMenu" name="view_menu">
    <property name="title">
     <string>Вид</string>
    </property>
    <addaction name="action_reset_slices"/>
    <addaction name="action_sync_with_first"/>
    <addaction name="action_sync_with_second"/>
    <addaction name="action_sync_with_third"/>
   </widget>
   <addaction name="main_menu"/>
   <addaction name="view_menu"/>
  </widget>
  <action name="action_open_dicom">
   <property name="text">
    <string>Открыть папку DICOM-файлами</string>
   </property>
  </action>
  <action name="action_sync_with_first">
   <property name="text">
    <string>Синхронизировать с первым срезом</string>
   </property>
  </action>
  <action name="action_sync_with_second">
   <property name="text">
    <string>Синхронизировать со вторым срезом</string>
   </property>
  </action>
  <action name="action_sync_with_third">
   <property name="text">
    <string>Синхронизировать с третьим срезом</string>
   </property>
  </action>
  <action name="action_reset_slices">
   <property name="text">
    <string>Сбросить срезы</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>SliceLabel</class>
   <extends>QLabel</extends>
   <header>uis.custom_widgets</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
